src/assets/styles/tools/mixins.scss

```diff
@@ -1,34 +1,16 @@
-/* stylelint-disable declaration-no-important */
-
-/* ================ Mixins ================ */
+// stylelint-disable declaration-no-important
```

Comment:
I'm curious. Can you make a test and see if it works with those comment style? I'm checking stylelint documentation and it looks like they use `/* */` one : https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md

Reply:
I had checked the documentation too and it works okay :') can you try on your side just to make sure?

Reply:
It works fine on my end as well, thanks! 👍